src/index.js

```diff
@@ -1,32 +1,18 @@
-const express = require('express'),
-  bodyParser = require('body-parser'),
-  session = require('express-session'),
-  cors = require('cors'),
-  app = express();
+import express from 'express';
+import bodyParser from 'body-parser';
+import setupSwagger from './utils/swaggerDefinition';
 
-app.use(cors());
-
-// Normal express config defaults
-app.use(require('morgan')('dev'));
+const app = express();
 
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
-app.use(require('method-override')());
```

Comment:
nice work here. however, remove these libraries from `package.json` also e.g `method-override` `morgan` etc.. it will help us all

Reply:
great! let me do it!